sdks/java/core/src/main/java/org/apache/beam/sdk/util/UnboundedScheduledExecutorService.java

```diff
@@ -201,9 +201,26 @@ public UnboundedScheduledExecutorService() {
         new ThreadPoolExecutor(
             0,
             Integer.MAX_VALUE, // Allow an unlimited number of re-usable threads.
-            Long.MAX_VALUE,
```

Comment:
Note that these were the original parameters when we didn't have a scheduled executor service.

Reply:
Agreed, I'm not sure the change to the service made this any different, likely could have been issue before too.

Reply:
Yeah, I think this will address a likely slow leak we have always had.